build-all.sh

```diff
@@ -48,6 +48,7 @@
 #                  [--disable-werror | --no-disable-werror]
 #                  [--strip | --no-strip]
 #                  [--release-name <release>]
+#                  [--with-tls | --without-tls]
```

Comment:
I would like this to be `--tls | --no-tls ` to maintain consistency with other options. It would be much better to use standart `--enable-*|--disable-*` however that should be done for all options then. Perhaps I should rename options for this release, but as of now, for consistency it should be `--tls|--no-tls`.